src/test/scala/cromwell/StdoutStderrWorkflowSpec.scala

```diff
@@ -17,16 +17,16 @@ class StdoutStderrWorkflowSpec extends CromwellTestkitSpec("StdoutStderrWorkflow
         eventFilter = EventFilter.info(pattern = s"persisting status of hello.hello to Done", occurrences = 1),
         fqn = "hello.hello",
         index = None,
-        stdout = Some("Hello world!\n"),
-        stderr = Some("")
+        stdout = Some(Seq("Hello world!\n")),
+        stderr = Some(Seq(""))
```

Comment:
It looks like we're changing the shape of the responses in our `logs` APIs, that needs to be communicated to our Blue friends.